### DISCUSSION TITLE
Software Implementations

### DISCUSSION THREAD

**Discussion by Rabioli** [beakerbrowser/beaker#1806] [General]
So I have been getting lately into p2p software since in my eyes is the only way for a truly free internet. I have encountered software that can make use of beaker or come as a default, like [Jami](https://git.jami.net/savoirfairelinux/ring-project) for p2p messaging, [LBRY](https://github.com/lbryio) as an alternative for youtube, [scuttlebutt](https://github.com/ssbc) for a p2p social network, all of that software would be great to have them in beaker out of the box, I don't really know where else to do a suggestion so here I am.

Aditionally on my own I have been trying to set up [write freely](https://github.com/writeas/writefreely) as an alternative for the default Blogger App template, that comes by default in beaker. I know a thing or two about programming but I am not really into it, in other words I can understand code but hardly develop it on my own right now, I would appreciate some help on that if you are able. What I have managed so far is to do a local implementation using mysql and writefreely but the only way I can actually run the software is through localhost, my current idea is to just change the http addres for the hyperdrive and see what happens, I'll keep you updated if I do manage to make it work. Write freely too is software that would be great for having out of the box instead of the current blog page.

**Comment by ghost** [comment]
the write-freely you referring to, is part of fediverse/activitypub – another, _decentralized_ concept, which is still requires servers, and not directly compatible with p2p at the moment

you can still serve it's static output as a beaker-drive tho

**Reply by Rabioli** [reply]
Yeah I just checked the site and I found [FediBlog](https://framagit.org/DavidLibeau/FediBlog) which is kind of a middle ground between write freely and the default blogging software on beaker, this one doesn't use a database so its way more portable I guess, the thing is that it does use php and I don't manage to run that in beaker, even tho in the git it does say it has some p2p support do you have any suggestion?
